deepset_cloud_sdk/_service/files_service.py

```diff
@@ -388,3 +390,52 @@ async def list_all(
             after_value = response.data[-1].created_at
             after_file_id = response.data[-1].file_id
             yield response.data
+
+    async def list_upload_sessions(
+        self,
+        workspace_name: str,
+        is_expired: Optional[bool] = False,
+        batch_size: int = 100,
+        timeout_s: int = 20,
+    ) -> AsyncGenerator[List[UploadSessionDetail], None]:  # noqa: F821
+        """List all upload sessions files in a workspace.
+
+        Returns an async generator that yields lists of files. The generator is finished when all files are listed.
+        You can specify the batch size per number of returned files using `batch_size`.
+
+        :param workspace_name: Name of the workspace whose files you want to list.
+        :param batch_size: Number of files to return per request.
+        :param timeout_s: Timeout in seconds for the listing.
+        :raises TimeoutError: If the listing takes longer than timeout_s.
+        """
+        start = time.time()
+        has_more = True
+
+        page_number: int = 1
+        while has_more:
+            if time.time() - start > timeout_s:
```

Comment:
I don't think we need this check. I left my terminal open on the list sessions command, had lunch, and came back expecting it to still list the next available sessions but got a timeout error instead. This was my face:
![pikachu_surprised_loop](https://github.com/deepset-ai/deepset-cloud-sdk/assets/7603827/1a0acd55-08ad-4674-b442-9bb597a27073)

Reply:
got what you mean.  With this check wanted to prevent a non terminating programm if someone calls 
`list(list_upload_sessions)`. So i added the timeout here. What do you think if we just increase it ?

Reply:
We could do that - or we could close the cli also after the timeout has been exceeded. Whatever makes more sense to you.

Reply:
done